app/models/ems_refresh/save_inventory_physical_infra.rb

```diff
@@ -2,6 +2,8 @@
 # Calling order for EmsPhysicalInfra:
 # - ems
 #   - physical_servers
+#   - physical_racks
+#     -physical_servers
```

Comment:
we shouldn't be saving physical_servers as children of the racks collection anymore